### PR TITLE
[READY] VR mind key transfer

### PR DESCRIPTION
**Pull request by DaxDupont**
🆑 Dax Dupont
fix: VR minds are now properly isolated.
config: VR maps have their own subdirectory now.
/🆑

### CODE REVIEW

File: code/modules/VR/vr_human.dm [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 /mob/living/carbon/human/virtual_reality
 	var/mob/living/carbon/human/real_me //The human controlling us, can be any human (including virtual ones... inception...)
+	var/datum/mind/real_mind
```
Comment (optimumtact) [inline code/modules/VR/vr_human.dm]: in an ideal case, you would simply track the source mob from the mind, that way if the real mind gets mindswapped, or some other crazy shit happens, you will track damage the actual appropriate real mob

and yes that could happen if I am guessing right

Comment (optimumtact) [reply code/modules/VR/vr_human.dm]: i.e mind.current

File: code/modules/VR/vr_sleeper.dm [hunk 2]
```diff
@@ -169,7 +171,7 @@
 			var/datum/outfit/O = new outfit()
 			O.equip(vr_human)
 		if(transfer && H.mind)
-			H.mind.transfer_to(vr_human)
+			vr_human.mind.key = H.mind.key
```
Comment (optimumtact) [inline code/modules/VR/vr_sleeper.dm]: you just set the key on the mob, not the mind sorry

Comment (optimumtact) [reply code/modules/VR/vr_sleeper.dm]: the rest should be handled by the login/logout procs

**Comment by AlexTheSysop42** [comment]
3 things:
1. IC, it makes sense for the VR human to have the same mind as the real one - they are, after all, controlling it directly.
2. Most other teleport methods already have an away mission check on both the source and destination, adding one to the problematic spells probably wouldn't be that much of an issue if you use that instead of a VR check. Examples: the cult teleport spell and rune and the hand tele.
3. If you do use this method, include a `mind` var on the vr sleeper so admins can make one that does transfer your mind.

**Comment by davethwave** [comment]
Can't wait for this to get merged!

**Comment by DaxDupont** [comment]
@optimumtact if you can check this and merge this when you have a moment I'd be happy so people stop nagging me about VR.

**Comment by DaxDupont** [comment]
@Cyberboss it seems changelogs weren't generated for this PR, can you check?

**Pull request merged**